packages/neuron-wallet/.env

```diff
@@ -5,6 +5,12 @@ MAINNET_SUDT_DEP_TYPE=code
 MAINNET_SUDT_SCRIPT_CODEHASH=0x5e7a36a77e68eecc013dfa2fe6a23f3b6c344b04005808694ae6dd45eea4cfd5
 MAINNET_SUDT_SCRIPT_HASHTYPE=type
 
+MAINNET_SUDT_INFO_DEP_TXHASH=0x81eeaaedc2909faf471cc17f8aeb66dd5e78d50ad1b7eb56e41ab821ee356330
```

Comment:
It would be helpful to add a TODO comment here, so we know these configure will need to be updated once `cell info script` has been deployed on mainnet.

Reply:
fix